views/lexicon/tables/datatablex.html.twig

```diff
@@ -6,14 +6,14 @@
             <tr>
                 <td class="table-responsive"></td>
                 <td class="table-selection"></td>
-                <th><a href="#">Name datatable</a></th>
-                <th><a href="#">IP</a></th>
-                <th><a href="#">Status</a></th>
-                <th><a href="#">Uptime</a></th>
-                <th><a href="#">Image</a></th>
-                <th><a href="#">RAM</a></th>
-                <th class="centered"><a href="#">CPUs</a></th>
-                <th><a href="#">Actions</a></th>
+                <th><a href="#name">Name</a></th>
```

Comment:
Should we be using links at all here, they don't go anywhere. Consider a button with an aria-label along the lines of aria-label="Name: activate to toggle column sorting.